modules/merquryfk/katcomp/main.nf

```diff
@@ -0,0 +1,45 @@
+process MERQURYFK_KATCOMP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    if (params.enable_conda) {
+        error "Conda environments cannot be used when using the FastK tool. Please use docker or singularity containers."
+    }
+    container 'ghcr.io/nbisweden/fastk_genescopefk_merquryfk:1.0'
+
+    input:
+    tuple val(meta), path(fastk1_hist), path(fastk1_ktab), path(fastk2_hist), path(fastk2_ktab)
+
+    output:
+    tuple val(meta), path("*.fi.png"), emit: filled_png , optional: true
+    tuple val(meta), path("*.ln.png"), emit: line_png   , optional: true
+    tuple val(meta), path("*.st.png"), emit: stacked_png, optional: true
+    tuple val(meta), path("*.fi.pdf"), emit: filled_pdf , optional: true
+    tuple val(meta), path("*.ln.pdf"), emit: line_pdf   , optional: true
+    tuple val(meta), path("*.st.pdf"), emit: stacked_pdf, optional: true
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def FASTK_VERSION = 'f18a4e6d2207539f7b84461daebc54530a9559b0'
+    def MERQURY_VERSION = '8f3ab706e4cf4d7b7d1dfe5739859e3ebd26c494'
```

Comment:
Is it standard to put this here now? The example in the guidlines put it outside the process block: https://github.com/nf-core/modules/blob/master/modules/homer/annotatepeaks/main.nf

Reply:
I can change them, but I prefer to have the versions within scope and as close to where they're being used. So far I've used this way with all merquryfk, genescopefk, and fastk modules. Is the homer way annotated as a standard? I feel like it's a throwback to when there were bits still outside the process like the options declaration.

Reply:
I'll merge for now, but if this should change then I'll make a pull request to change them all

Reply:
Sounds good. I just checked against all the modules and I'm the only one that did it differently.